policies/design-process.md

```diff
@@ -1,160 +1,164 @@
+policy adopts a graded system of review in which the degree of formal approval
+by the OTC which a design must undergo is proportionate to the scope and risks
+of the design. For example, a design which involves new or evolved public APIs
+may require a greater amount of scrutiny, whereas a purely internal design may
+require that the OTC simply be notified of the design document and invited to
+comment, with a presumption of approval in the absence of objections.
+
+Requirement for Design Documents
+--------------------------------
+
+A design document is required for any proposed enhancement which adds new APIs
+or evolves or modifies existing APIs.
+
+For any other kind of proposed enhancement, a design document should be created
+if it incorporates design decisions or aspects significant enough to warrant
+one. For example, if an enhancement adds a new internal module with clearly
+delineated boundaries with a documented internal API which can be consumed by
+other code internal to OpenSSL, a design document is desirable. This example is
+not exhaustive. The proposer may use their discretion in determining whether a
+design document is desirable, but any OTC member may require that a design
+document be produced.
+
+Levels of Scrutiny
+------------------
+
+There are three levels of scrutiny which can be applied to a design, listed
+below in ascending order of severity:
+
+ - Notify
+ - Present
+ - Approve
+
+These levels work as follows:
+
+ - At the **Notify** level, the OTC is notified of a new design
+   when it is available for review, via an email to the OTC mailing list.
+   OTC members and committers can review and comment on the design.
+   A minimum waiting time of one week applies to ensure OTC members have the
+   opportunity to review the design.
+
+ - At the **Present** level, the OTC is notified of a new design
+   in the same way that it is at the Notify level. The same minimum waiting time
+   applies. The design is also introduced and explained in a presentation given
+   by its presenter to the OTC in a meeting of the OTC. The OTC has
+   opportunities to ask questions and raise concerns at this meeting.
+
+ - At the **Approve** level, the OTC must explicitly approve the design
+   by making a decision as the OTC. The OTC should be notified of a
+   new design in the same way they are notified at the Notify level.
+   A presentation to the OTC may be made but is not required.
+
+When a design is produced, it should be submitted to the OpenSSL repository as a
+PR. It should be determined which level of scrutiny is appropriate according to
+this policy and this should be noted in the PR. Any applicable actions (such as
+notifying the OTC via the list) should be carried out once the design is ready
+for review.
+
+Any OTC member may object to the processing of a design at a given level of
+scrutiny. In this case, the next highest level of scrutiny is used. An OTC
```

Comment:
s/next highest level/next higher level/

Reply:
I think this should be rewritten to say that any OTC member can object and request processing at any higher level.

Being able to go to the next higher or the top level is effectively the same thing but more wordy and less streamlined IMO.  It's also less future proof if we ever add more review levels (which I really hope never happens).